quantum/quantum_access.rst

```diff
@@ -0,0 +1,185 @@
+============================================
+
+Welcome! The information below introduces how we structure projects and user accounts for 
+access to the Quantum resources within the QCUP program. In general, OLCF QCUP resources 
+are granted to projects, which are in turn made available to the approved users associated 
+with each project. 
+
+To gain access, you must first submit a project proposal to the OLCF QCUP. The Quantum Resource 
+Utilization Council (QRUC), as well as independent referees, review and approve all QCUP 
+project proposals. Applications to QCUP are accepted year-round via the project application form 
+found below. Once your project is approved, then all of the users associated with the 
+
+QCUP Priorities
+***************
+
+There are several broad aims of the Quantum Computing User Program at OLCF, which are as follows:
+
+1. **Enable Research**
+
+The QCUP aims to provide a broad spectrum of user access to the best available quantum computing 
+systems. Once a user’s intended research has been reviewed for merit and user agreements have been 
+established, we seek to provide users with the opportunity to become familiar with the unique 
+aspects and challenges of quantum computing, as well as to implement and test quantum algorithms 
+on the available systems.
+
+2. **Evaluate Technology**
+
+The QCUP aims to aid in the evaluation of technology by monitoring the breadth and performance 
+of early quantum computing applications. How users integrate quantum computing with scientific 
+computing is a question constrained by both application, infrastructure constraints, and the 
+use cases expected for the associated computational system. Through the QCUP program, users 
+can explore new potential computational research applications, and potentially accelerate 
+existing scientific applications using quantum processors and architectures. Research projects 
+supported include advanced scientific computing, basic energy science, biological environmental 
+research, high-energy physics, fusion energy science, and nuclear physics, among others.
+
+3. **Engage the Community**  
+
+The QCUP aims to engage the quantum computing community and support the growth of the quantum 
+information science ecosystems. Our quantum computing users range in quantum computing experience 
+from novice to expert; users are from US national labs, universities, government, and industry. 
+User groups utilize quantum computing expertise to investigate diverse application interests, 
+using multiple programming languages, quantum-classical programming, and multiple software 
+environments. Most projects focus on proof-of-principle demonstrations and/or new method 
+development. Some projects focus on application performance and/or benchmarking, and 
+additionally some projects focus on device characterization, verification, and validation.
+
+
+
+Request a New Project Allocation
+******************************
```

Comment:
Need two more "*", it needs to match the heading length

Reply:
fixed